### DISCUSSION TITLE
Mention organization in comments

### DISCUSSION THREAD

**Discussion by rhymes** [forem/forem#15897] [Icebox]
**Is your feature request related to a problem? Please describe.**

As it's possible to mention users in comments whom will receive a notification for the mention, it would be nice if it was possible to mention organizations too, maybe with the admin of the organization receiving such notification.

**Describe the solution you'd like**

If I type @devteam or any other org in a comment, the `@devteam` should become a link like when I mention any other user and the organization admin should receive the comment as a notification.

**Comment by bolariin** [comment]
if both an admin and the organization are mentioned in a comment, should this create one mention or two?

**Comment by Zhao-Andy** [comment]
I think in this case it makes sense to notify _just_ the organization, and create a single notification for that org. The notification model can take either a `user_id` or an `organization_id` and **not both**.

**Comment by JennieOcken** [comment]
This is a great idea but not in our current focus / priority area.  I am going to throw it in the icebox.  We regularly review those ideas when building our roadmap.

**Comment by dime421** [comment]
A feature request is a comment, message[,](https://gomcraft.com/minecraft-pocket-edition-1-19-apk/) or asks from a user to implement a specific feature into your product. Feature requests can be used to decide how to iterate on an existing product.
